cmd/prometheus/main.go

```diff
@@ -345,6 +342,9 @@ func main() {
 	a.Flag("web.remote-write-receiver.accepted-protobuf-messages", fmt.Sprintf("List of the remote write protobuf messages to accept when receiving the remote writes. Supported values: %v", supportedRemoteWriteProtoMsgs.String())).
 		Default(supportedRemoteWriteProtoMsgs.Strings()...).SetValue(rwProtoMsgFlagValue(&cfg.web.AcceptRemoteWriteProtoMsgs))
 
+	a.Flag("web.enable-otlp-receiver", "Enable API endpoint accepting OTLP write requests.").
```

Comment:
```suggestion
	a.Flag("web.otlp-receiver", "Enable API endpoint accepting OTLP write requests.").
```

Just to be consistent with remote-write. The flag there is `web.remote-write-receiver`, right?

Reply:
Yes it already is, check https://github.com/prometheus/prometheus/pull/14894/files#diff-6554f0eaaf28d34fad1ec0ad13393db25deb69c649b377839173b87c946aae9cR336-R339

Reply:
ah, sorry about that... I don't know why the remote write flag was called `web.remote-write-receiver` 🤨